__tests__/server/question-routes.spec.js

```diff
@@ -0,0 +1,21 @@
+const supertest = require('supertest');
+const app = require('../../server/server');
+const request = supertest(app);
+
+describe('GET /api/questions', () => {
+  it('should print questions in json format when request is successful', async () => {
+    const res = await request.get('/api/questions');
+    // The current bruteforce way of checking it is to see if there are more than 8 questions returned.
+    // TODO: should change the following after we implement the create question function.

```

Comment:
This is actually OK because we won't be making a create question function. If any questions are added, they will be added directly to AirTable by our client. So checking > 8 is actually OK, especially with the tests I describe below. So you can get rid of the TODO.